src/util/Util.ts

```diff
@@ -83,6 +83,32 @@ export const pipeSafe = <T extends Writable>(readable: NodeJS.ReadableStream, de
   return destination;
 };
 
+/**
+ * Makes sure that listeners always receive the error event of a stream,
+ * even if it was thrown before the listener was attached.
+ * @param stream - Stream that can potentially throw an error.
+ *
+ * @returns The wrapped stream.
+ */
+export const wrapStreamError = <T extends NodeJS.EventEmitter>(stream: T): T => {
+  let error: Error | undefined;
+  let timeout: NodeJS.Timeout;
+  stream.on('error', (err): void => {
+    error = err;
+    timeout = setTimeout((): void => {
+      logger.warn(`No listener was attached but error was thrown: ${err.message}`);
```

Comment:
For debugging purposes, it might be useful to include a trace where the stream was created, or some other form of identifier.